packages/rich-text/src/plugins/Hyperlink/EntityHyperlinkTootip.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { hyperlinkTooltipStyles as styles } from './styles';
-import { Tag } from '@contentful/forma-36-react-components';
+import { EntityStatusBadge } from "@contentful/f36-badge";
```

Comment:
This is kind of off-topic, but is there a reason for `EntityStatusBadge` not being documented in the website and storybook?

Reply:
I don't really know why EntityStatusBadge is not documented. I will write this down as something that we need to impove.